script/Arcueid/Day 8/08_00_ARC08_3.txt

```diff
@@ -145,7 +145,7 @@ Only twenty more minutes until the end of fourth period.
 [sha:a52eda6ec9c0408035dbd699aa071c853df59e6c]{
 -- Page 147, Offset 20038.
 -- 今日の授業は五時限目までだ。
-Classes let out after fifth period today.
+Classes finish after fifth period today. 
```

Comment:
Let out feels more vernacular to me, but possible it's a britishism.
Also no trailing spaces!